### ISSUE TITLE
Should mix and the color-bin-ops be documented?

### ISSUE THREAD

**Issue by brunchboy** [jolby/colors#6]
I ended up rolling my own poor mix function last night because I did not see anything like that in the API reference, but just noticed after submitting my pull request for the RGB creation fix that there had been another pull request fixing a `mix` function, so I dug into the source, and found it and `color-add`, `color-mult`, et. al. Those look really useful! Could they be added to the API reference?

**Comment by jolby** [comment]
They should be added to the reference, I can't remember why they weren't in the first place. I'll be happy to apply any patch you send or I can do it next time I do a maintenance pass on the project (might be a while! )

**Comment by brunchboy** [comment]
Ah! I did not realize that was generated from the project; since it is hosted on a different site I did not know I could contribute to it. Yes, I will try to figure out how to get these added; since I refer prominently to your documentation in my own, https://github.com/brunchboy/afterglow/wiki/Working-with-Color  (so I don’t have to try to document your large and useful library myself), I would like people to be able to find these!
